testing/go/functions_test.go

```diff
@@ -263,6 +274,7 @@ func TestSystemInformationFunctions(t *testing.T) {
 			Assertions: []ScriptTestAssertion{
 				{
 					Query: `SELECT current_catalog;`,
+					Cols:  []string{"current_database"}, // TODO: Should be current_catalog
```

Comment:
This should be a skipped test with the correct output, rather than a test enforcing the wrong output.